src/Sylius/Bundle/ProductBundle/Controller/ProductAttributeController.php

```diff
@@ -86,17 +86,43 @@ public function renderAttributeValueFormsAction(Request $request): Response
      */
     protected function getAttributeFormsInAllLocales(AttributeInterface $attribute, array $localeCodes): array
     {
-        $attributeForm = $this->get('sylius.form_registry.attribute_type')->get($attribute->getType(), 'default');
+        $attributeForm = $this
+            ->get('sylius.form_registry.attribute_type')
+            ->get($attribute->getType(), 'default')
+        ;
 
         $forms = [];
+
+        if (!$attribute->isTranslatable()) {
+            array_push($localeCodes, null);
+
+            return $this->createFormAndView($attributeForm, $attribute, null, $forms);
+        }
```

Comment:
Bigger problem that I was in this PR is handling `__not_translatable__` key, its not possible without overwritten few class with validation from SyliusAttribute, SyliusBundle and ChoiceToValueTransformer form Symfony, but `null` variable is handling on backend site.

`Null` is problem with `tab` library from Semantic UI because `null` is assigning to `data-tab` and `data-tab` with null variable is not handling by 'tab'. Replacing `null` with ``__not_translatable__`` on frontend site solving a problem.

<img width="1146" alt="Screenshot 2020-10-28 at 20 05 04" src="https://user-images.githubusercontent.com/39232096/97624965-295e6c00-1a28-11eb-9ad3-8921d2282117.png">